C-Plus-Plus/other/Minimum_number_of_swaps_for_bracket_balancing.cpp

```diff
@@ -0,0 +1,90 @@
+/*
+
+We are given a string of 2N characters
```

Comment:
take all this at the bottom of code file. While there is no particular reason to do so. But as a personal liking I try to keep all the most important things in a code file at the top or the most reachable point. all the information or not so important pieces hidden in the bottom.